src/components/ContainerQuery/ContainerQuery.test.tsx

```diff
@@ -0,0 +1,47 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import ContainerQuery from '.';
+import { create as createRenderer, ReactTestRendererJSON } from 'react-test-renderer';
```

Comment:
Why not use enzyme like the other test files? Just curious

Reply:
We also seem to be doing this for `Clickable`, not sure why.

Reply:
Yeah there's no need to be doing this, we should move to enzyme. (This was copied from the clickable test)